### DISCUSSION TITLE
Stackstorm webui stopped working

### DISCUSSION THREAD

**Discussion by patsuraj** [StackStorm/st2#5393] [Q&A]
Hello everyone, I am new to stackstorm and just inherited our stackstorm server at work as a team member recently left. I am having an issue that I am trying to figure out. The server stopped working and I rebooted it which started everything up again. The only issue is that the webUI is not working anymore. The firewall rules are already in place to be able to access it. What happens now is when I go to the webUI the browser gives me a "This site can't be reached" message. on the server itself within CLI I checked the status of the st2web service and it says "Loaded: not-found (Reason: No such file or directory) Active: inactive (dead)" 

Is there anyone that can help me with this issue? Thank you in advance!

**Comment by minsis** [comment]
Is nginx running?

**Reply by patsuraj** [reply]
I have just checked the status of nginx and looks like its in a failed state. The error states "host not found in upstream..."

**Reply by minsis** [reply]
The st2web runs in nginx as the web server. Sounds like you need to maybe troubleshoot why nginx isn't working.

**Reply by patsuraj** [reply]
I ended up solving the issue! Thank you! 

I am having another issue as well if by any chance you would know a solution. If not, it not related to this issue so I'll create a new thread. The issue I am having is: I have an automation to pull in new tickets into the Hive which works fine. Once the ticket is worked on in the hive and closed out it will then export the comments/notes on the ticket into the original ticket service and close it out there. The later step is not working due to the action in the workflow having a timeout duration. The time it takes to complete a ticket varies on team member and other factors like when they can get to the ticket or what the ticket needs done. Im not sure if this is the best way to go about this but would changing the timeout duration to an infinite time so that the automation waits until it sees the ticket is closed out and then it can continue and complete the automation work? Or is there another more efficient way you would know?

Thanks in advance!

**Reply by minsis** [reply]
Create another post so maybe others can chime in on ideas.

I will give my suggestion here though: 
1. I wouldn't set an infinite timeout - if thats even possible - as you're now tying up a thread. If say you're limited to 50 threads and you're now waiting on 100 tickets no other processes are going to run until the queue clears out.
2. You'd be better off trying to figure out how to create a custom sensor to overcome this. Then from there you can create an action to trigger either  a single action or a workflow of actions to get the info of the ticket and then close it out.
